appveyor.yml

```diff
@@ -6,15 +6,22 @@ branches:
 
 cache:
   # Note that we don't use cache for local Maven repository - see https://github.com/SonarSource/sonar-java/pull/525
-  - C:\ProgramData\chocolatey\bin -> appveyor.yml
-  - C:\ProgramData\chocolatey\lib -> appveyor.yml
-  - C:\bin\apache-maven-3.2.5 -> appveyor.yml
+  - C:\Users\appveyor\maven -> appveyor.yml
```

Comment:
@dbolkensteyn if this caching of Maven was introduced to overcome instability of Chocolatey, then this didn't worked - see description of PR: Chocolatey anyway always was trying to install Maven.

The main point here is to remove Chocolatey, primarily because of instability of its public feed, and the rest was preserved as it was. Note that it is even recommended to use own feed -  https://github.com/chocolatey/choco/wiki/ChocolateyFAQs#should-my-organization-depend-on-use-the-community-feed-httpschocolateyorgpackages
So whether caching of Maven helps a lot or not is IMO another story, but this can be easily measured by timing Appveyor phases.